client/src/components/Recommendation.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react'
+import { Tab } from '@headlessui/react'
+import MovieShow from './MovieShow'
+
+function classNames(...classes) {
+	return classes.filter(Boolean).join(' ')
+}
```

Comment:
What is this for? Why this is needed?

Reply:
I see you're using it to do different styles on selection. There's a better approach. Declare a function named `getSelectionStyles(isSelected=false)` or similar. That function should conditionally render all the class name strings. Finally, you can do `<Component classNames={getSelectionStyles(isSelected)} />

Reply:
to join classes for selected condition, check line 112.